templates/book_borrowing_dashboard.html

```diff
@@ -1,7 +1,13 @@
 {% extends "base.html" %}
 {% block title %}Contact{% endblock %}
 {% block content %}
+{% from "macros.html" import table_row, copy_info_row, flashed_message with context %}
```

Comment:
I think we can delete here `table_row, copy_info_row` macros and leave only `flashed_message` :)

Reply:
Nice catch.